Movies Time/Libraries/NetworkProvider.swift

```diff
@@ -117,7 +138,8 @@ final class NetworkProvider {
         
         switch requestType {
         case .movieDetails:
-            break
+            queryParams["api_key"] = token
```

Comment:
if the params are the same for both request types, you can put them in the same case so you don't duplicate code